modules/serverless-autoscaling-minscale-kn.adoc

```diff
@@ -5,7 +5,7 @@ You can use the `kn service` command with the `--min-scale` flag to create or mo
 
```

Comment:
Same as above. If time permits, please comment which assemblies this module is included in.